packages/components/ng-ovh-payment-method/src/components/choice/template.html

```diff
@@ -42,7 +42,7 @@
     <oui-radio
         data-name="paymentMethod"
         data-model="$ctrl.model"
-        data-value="false"
+        data-value="null"
```

Comment:
I think no need to use data- for name, value and required as they are standard HTML attributes

Reply:
You're right !

But `ui-kit` overrides some default `html` attributes, this is why it's needed :(